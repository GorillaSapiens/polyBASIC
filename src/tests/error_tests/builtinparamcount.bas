a = sin(10,20)
