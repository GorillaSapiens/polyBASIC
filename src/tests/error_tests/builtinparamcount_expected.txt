ERROR: @1:5, INCORRECT NUMBER OF PARAMETERS ❮SIN❯ 2<>1
